clustered/server/service/build.gradle

```diff
@@ -22,6 +22,8 @@ plugins {
 dependencies {
   service project(':clustered:server:service-api')
   service "org.terracotta:offheap-resource:$terracottaPlatformVersion"
+  service "org.terracotta:statistics:$statisticVersion"
+  service "org.terracotta.common:common-structures:$terracottaPlatformVersion"
```

Comment:
@chrisdennis @saurabhagas : structures should come transitively from offheap jar. There is a bug in tc-platform. I will raise a PR and the last line could be removed.

For statistics, this is another matter, it can be left as-is as service right now.

Reply:
Updated the PR with release 5.7.4-pre10 to remove the second line